src/ts/settlement.ts

```diff
@@ -6,6 +6,7 @@ import {
   ethers,
 } from "ethers";
 
+import { Interaction } from "./interaction";
```

Comment:
Small change that makes it easier to perform tree shaking (i.e. reducing the code that makes it into the final bundle) when building for the FE.